app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -90,203 +88,84 @@ For safeguarding reasons, schools will assess your suitability to teach from a r
 
 Teacher training providers will ask for:
 
-* evidence that applicants for teacher training have a bachelor’s degree from a university (non-UK or in the UK)
+* evidence you have a bachelor’s degree from a university (non-UK or in the UK)
 * a criminal record check
 
 You can apply to the [Ministry of Internal Affairs of Ukraine](https://vytiah.mvs.gov.ua/app/landing) for a criminal record check, which will be emailed to you. You can then contact the [Ukrainian Embassy in the UK](https://uk.mfa.gov.ua/en) for a translation of the criminal record check and a letter confirming its validity.
 
-We understand it might not be possible to provide all this evidence. However, it’s helpful to show schools and training providers you have made every effort to do so. They can then reach a decision about your suitability, looking at the evidence available. 
-
-### Help proving your professional qualification to training providers and employers
+### Your professional qualifications
 
-UK ENIC can give you a [statement of comparability](https://www.enic.org.uk/Qualifications/SOC/Default.aspx) showing how your academic and professional qualifications compare to their English equivalent. However, you will need scans or photographs of your original documents. If you are applying to train to teach, you can [get your statement of comparability for free](#get-help-with-international-qualifications).
+UK ENIC can give you a [statement of comparability](https://www.enic.org.uk/Qualifications/SOC/Default.aspx) showing how your academic and professional qualifications compare to their English equivalent. However, you will need scans or photographs of your original documents. If you are applying to train to teach, you can [get your statement of comparability for free](/non-uk-teachers/train-to-teach-in-england-as-an-international-student/check-your-eligibility).
```

Comment:
Change the URL from:
`/non-uk-teachers/train-to-teach-in-england-as-an-international-student/check-your-eligibility`
to:
`/non-uk-teachers/train-to-teach-in-england-as-an-international-student#check-your-eligibility`

(last `/` becomes a `#` to indicate a page anchor)

Reply:
thank you!!!

Reply:
@laurahermionetennant92 as the GIT website focus is on training to teach should these 2 sentences be the other way round so we talk about applying to train to teach first and then if you're already a teacher?

Reply:
@laurahermionetennant92 you'll and you're

Reply:
I have swapped the sections to Train to teach comes first :)
Have left the ENIC sentence because it works better as is...